content/4_Gold/BFS.mdx

```diff
@@ -148,6 +153,23 @@ deque.removeLast(); // [1, 3]
 
 </JavaSection>
 
+<PySection>
+
+#### Python
+
+In Python, `collections.deque()` is used for a deque data structure. The four methods for adding and removing are `appendleft`, `popleft`, `append`, and `pop`.
+```py
+d = collections.deque()
+d.appendleft(3); # [3]
```

Comment:
1. semicolons not needed
2. 2 spaces after line of code for inline comment

Reply:
right mb, forgot python doesnt use semicolons rip